spring-cloud-function-samples/function-sample-azure/README.adoc

```diff
@@ -1,132 +1,328 @@
-You can run this Azure function locally, similar to other Spring Cloud Function samples, however 
+== Running Locally  `
```

Comment:
I updated the docs a bit and also added details around the Docker image side of things. I also QA'ed them and followed them fresh to ensure they worked and I was able to deploy into Azure.